examples/flutter_example/lib/main.dart

```diff
@@ -55,12 +55,19 @@ class _MyHomePageState extends State<MyHomePage> {
   @override
   void initState() {
     super.initState();
-    One.initializeOne(SITE_KEY, TOUCHPOINT, API_KEY, SHARED_SECRET, USER_ID, HOST, false);
-    One.setThunderheadLogLevel(true);
+    // One.optOut(true);
 
     // Calling this in `initState()` for demonstration purposes.
-    // When to send these Interaction requests should be aligned with your business use cases rather than follow exactly this code placement.
-    One.sendInteraction("/home", null);
+    One.initializeOne(SITE_KEY, TOUCHPOINT, API_KEY, SHARED_SECRET, USER_ID, HOST, false).then((_) {
+      One.setThunderheadLogLevel(true);
```

Comment:
"set logel level true" doesn't read well.  Please create a follow up ticket to review these apis in detail as I am just coming on the project.

Reply:
https://thunderhead.jira.com/browse/MOB-1907